Automattic/wp-calypso
# [e2e] Disable ensureSaved call for "Private Posts"

#### Changes proposed in this Pull Request

PR https://github.com/Automattic/wp-calypso/pull/33417 didn't make any difference. Test `Calypso Gutenberg Editor: Posts (mobile) Private Posts:` is still failing with the same [error](https://circleci.com/gh/Automattic/wp-calypso/306996). With disabled `ensureSaved` call this error shouldn't happen anymore. 

#### Testing instructions

I couldn't reproduce this locally, it happens only on CircleCI. I've run tests multiple times and no error occurred. make sure that mobile tests are passing.
